examples/with-apollo-auth/pages/_app.js

```diff
@@ -1,9 +1,11 @@
 import App from 'next/app'
 import React from 'react'
-import { ApolloProvider } from 'react-apollo'
+import { ApolloProvider } from '@apollo/react-hooks'
 import withApollo from '../lib/withApollo'
 
 class MyApp extends App {
+  static displayName = 'MyApp'
```

Comment:
What's the point of the displayName variable? it does not use in _app.js

Reply:
It is needed for better devtools experience. Check how react devtools show it:
![image](https://user-images.githubusercontent.com/15915715/63256736-d9f7aa00-c280-11e9-8991-eb1e4c1eb25b.png)

Reply:
Nice, so do you agree to write a single line comment for it? like the comment that you mention above.

Reply:
Yes, sure